File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
       int prev = nums.at(nums.size()-1);
       int count = 0;

       for(int i = 0; i < nums.size(); i++){
           if(prev > nums[i]) count++;
           prev = nums[i];
       } 

       if(count > 1) return false;
       return true;
    }
};

File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var check = function(nums) {

    let prevEle = nums[nums.length - 1];
    let count = 0;
    
    for(let i = 0; i<nums.length; i++){
        const cEle = nums[i];
        if(prevEle > cEle) count++;
        prevEle = cEle;
    }

    if(count === 1) return true;
    else return false;
};